Cargo.toml

```diff
@@ -52,4 +52,4 @@ clap = { version = "4.0.0", default-features = false, features = ["std", "derive
 env_logger = "0.10.0"
 tracing = "0.1"
 tracing-subscriber = "0.3"
-tracing-log = "0.1.3"
+tracing-log = "0.1"
```

Comment:
nit: Why was this changed?

I've been slowly been making my version requirements more specific

Reply:
That was a mistake. I thought I made them specific, but it was the reverse lol.